### DISCUSSION TITLE
Bug/Help? Border Waypoints/Course Waypoints not appearing

### DISCUSSION THREAD

**Discussion by Conan2255** [Courseplay/Courseplay_FS22#3268] [Q&A] [answered]
Hi all, 
Just looking for some help with figuring this out, or if its a bug. 
I've read the old posts about the Eye not being for the waypoints, but rather using the three dots... I dont know if its outdated, or what, but I do not have those dots, and other waypoint/show course settings dont work.

Ill attach images and whatnot, but let me know if anything else is needed? 
![Farming Simulator 22 3_06_2024 4_20_07 PM](https://github.com/Courseplay/Courseplay_FS22/assets/171541123/c86b2831-c21b-4727-ac9e-1a33c3a16e12)
![Farming Simulator 22 3_06_2024 4_21_34 PM](https://github.com/Courseplay/Courseplay_FS22/assets/171541123/01831fe9-8526-4541-945f-9b855ca577cc)
![Farming Simulator 22 3_06_2024 4_21_50 PM](https://github.com/Courseplay/Courseplay_FS22/assets/171541123/25381d57-03bf-4267-b1b6-e89c6c9aae93)

https://github.com/Courseplay/Courseplay_FS22/assets/171541123/9130233d-18c6-4205-8921-d13ed06d25e1

**Comment by Conan2255** [comment]
I have uninstalled and reinstalled courseplay multiple times. 
I have uninstalled and reinstealled FS22 multiple times. 
I have tried on multiple maps.

**Reply by Tensuko** [reply, marked as answer]
reinstalling a mod never helps, thats nonsence.
reinstalling FS22 also won't help.

What you have to do:
Go into ..\Dokumente\My Games\FarmingSimulator2022\shader_cache and delete all files.
Your game will lag for a while and rebuild the shader, thats normal. (Like after a patch).

**Reply by Conan2255** [reply]
Thanks a million, worked instantly with little to no lag.
